cmd/restic/cmd_init.go

```diff
@@ -65,7 +65,7 @@ func runInit(opts InitOptions, gopts GlobalOptions, args []string) error {
 		return err
 	}
 
-	s := repository.New(be)
+	s := repository.New(be, gopts.MinPackSize*1024*1024)
```

Comment:
IMO no need to set the `MinPackSize` for the `init` command - there are no packs to be written here.

Reply:
MinPackSize is used on line 283, in `func (r *Repository) SaveAndEncrypt()`

It might not need to be set in the init command, but maybe a setter if not?

Reply:
Went with the setter, as discussed above.